WEB/Src/DependencyCollector/DependencyCollector/DependencyTrackingTelemetryModule.cs

```diff
@@ -282,10 +282,9 @@ protected virtual void Dispose(bool disposing)
         [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
         private static void PrepareFirstActivity()
         {
-            using (var activity = new Activity("Microsoft.ApplicationInights.Init"))
-            {
-                activity.Start();
-            }
+            var activity = new Activity("Microsoft.ApplicationInsights.Init");
+            activity.Start();
+            activity.Stop();
```

Comment:
Is this extra line of code related to the downgrade of DiagnosticSource?

Reply:
Yes. using dispose pattern is not supported in earlier version of DiagnosticSource.